src/main/scala/it/pagopa/interop/notifier/service/converters/package.scala

```diff
@@ -22,4 +24,11 @@ package object converters {
     type EventType = Value
     val ADDED, CREATED, DELETED, UPDATED, SUSPENDED, DEACTIVATED, ACTIVATED, ARCHIVED, WAITING_FOR_APPROVAL = Value
   }
+
+  def createMessageId(
```

Comment:
```suggestion
  def getMessageId(
```
as it does not create an object through `DynamoService`